README.md

```diff
@@ -19,6 +19,7 @@ At the time of joining, the HR potrays the company like it is match made in Heav
 | 10 | Usually help half or whole financially for IT events and certifications?|
 | 11 | Can you describe the culture of the company?|
 | 12 | How easily the upper management is approachable for suggestion or queries?|
+| xx | Does your organisation is open for on-site opputunities and Can I get this oppurtunity on starting of day of an organisation? |
```

Comment:
- their is a typo in spelling "opportunity", please fix that
- Second part of the question could be modified to `if yes, do you give these opportunity to new comers`?

Reply:
Thanks for the update. Changes are Done, Please Review it again